src/bin/pg_autoctl/cli_formation.c

```diff
@@ -350,10 +404,21 @@ keeper_cli_formation_create(int argc, char **argv)
 	FormationConfig config = formationOptions;
 	Monitor monitor = { 0 };
 
-	if (!monitor_init_from_pgsetup(&monitor, &config.pgSetup))
```

Comment:
Use `cli_monitor_init_from_option_or_config` instead?

Reply:
Can't, we have a `FormationConfig config` structure instance, not a `KeeperConfig config` one...